docs/reference/quickstart/run-elasticsearch-locally.asciidoc

```diff
@@ -125,6 +125,10 @@ The service is started with a trial license. The trial license enables all featu
 
 To connect to the {es} cluster from a language client, you can use basic authentication with the `elastic` username and the password you set in the environment variable.
 
+.*Expand* for details
```

Comment:
I dunno if I love this bolding - feels like all of it or none of it should be bolded

Reply:
I need a style guide refresh on some of this stuff, merging for now :)